server/pbench/bin/index-pbench

```diff
@@ -644,6 +685,28 @@ class ToolData(object):
             datafiles.append(datafile)
         return datafiles
 
+    @staticmethod
+    def get_json_files(handler, ftype, iteration, sample, host, tool, toolgroup, mdlog, tb):
+        """
+        Fetch the list of json files for this tool,and
+        return a list of dicts containing their metadata.
+        """
+        # import pdb; pdb.set_trace()
+        path = os.path.join(iteration, sample, "tools-%s" % (toolgroup,), host, tool, "json")
+        paths = [x for x in tb.getnames() if x.find(path) >= 0 and tb.getmember(x).isfile()]
+        datafiles = []
+        for p in paths:
+            fname = os.path.basename(p)
+            try:
```

Comment:
Get rid of ftype, hander_rec (anything not used in this particular incarnation of the indexer). Pass the handler record directly as needed to distinguish json from csv handling in make_source.